src/core_plugins/getting_started/public/components/getting_started/getting_started.js

```diff
@@ -0,0 +1,63 @@
+import { uiModules } from 'ui/modules';
+import uiChrome from 'ui/chrome';
+import 'ui/storage';
+import 'ui/filters/trust_as_html';
+import { gettingStartedRegistry } from 'ui/getting_started_registry';
+import { GETTING_STARTED_OPT_OUT } from '../../lib/constants';
+
+import kibanaLogo from 'ui/images/logo-kibana-small.svg';
+import beatsLogo from 'ui/images/logo-beats-small.svg';
+import logstashLogo from 'ui/images/logo-logstash-small.svg';
+import dashboardIcon from 'ui/images/icon-dashboard.svg';
+import shieldIcon from 'ui/images/icon-shield.svg';
+
+import template from './getting_started.html';
+import './getting_started.less';
+
+const app = uiModules.get('kibana');
+
+app.directive('gettingStarted', function ($injector) {
+
+  const localStorageService = $injector.get('localStorage');
+
+  return {
+    restrict: 'E',
+    template: template,
+    scope: {
+    },
+    bindToController: true,
+    controllerAs: 'gettingStarted',
+    controller: class GettingStartedController {
+      constructor() {
+        if (this.hasOptedOut) {
```

Comment:
There's a flash of content when you first load the getting started page. If you log `uiChrome.getVisible()` before this condition, it will be `true`. I think this is what's causing it.

I tested out a solution, and it seems to work. We just need to change https://github.com/elastic/kibana/blob/master/src/ui/public/chrome/directives/kbn_chrome.js#L40 to this:

```javascript
        // Hide the chrome by default, so it doesn't interfere with the Getting Started page.
        // Otherwise, show it if Kibana hasn't been embedded.
        const isChromeVisible =
          $location.search().embed === undefined
          ? false
          : !$location.search().embed;

        internals.setVisibleDefault(isChromeVisible);
```